### ISSUE TITLE
[Accessibility :Alternative content] Loading message in Software update grid does not have alternative content

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/gingerbase#68]
Loading message textarea in Software update grid does not have alternative content to convey it's purpose.
